.circleci/config.yml

```diff
@@ -74,147 +74,72 @@ jobs:
 
 
 
```

Comment:
There is no "wip" anymore in `thiccc` (it's moved to `legacy`) branch so I removed half of the tests.